zebra-chain/src/transaction/unmined.rs

```diff
@@ -0,0 +1,171 @@
+//! Unmined Zcash transaction identifiers and transactions.
+//!
+//! Transaction versions 1-4 are uniquely identified by narrow transaction IDs,
+//! whether they have been mined or not,
+//! so Zebra and the Zcash network protocol don't use wide transaction IDs for them.
+//!
```

Comment:
Minor: Would it make sense to mention that V5 transactions are different?

Reply:
Thanks, I didn't realise I'd left them out!

See 2a4b4e38f46d423748472b58869cb90cefd95ff9